### PR TITLE
Fixed inconsistent use of plurals Ref #22791

### PR DESCRIPTION
**Pull request by hardik-codes**
Pull Request for Issue #22791  

### Summary of Changes
Fixed the two items with (s) in System menu to all plural without any brackets.

Language(s) & Content Language(s) now appear as Languages and Content Languages respectively in System menu

### CODE REVIEW

**Comment by brianteeman** [comment]
I disagree with this change

File: administrator/language/en-GB/en-GB.mod_quickicon.ini [hunk 1]
```diff
@@ -15,7 +15,7 @@ MOD_QUICKICON_GLOBAL_CONFIGURATION="Global"
 MOD_QUICKICON_GROUP_DESC="The group of this module (this value is compared with the group value used in <strong>Quick Icons</strong> plugins to inject icons). The 'mod_quickicon' group always displays the Joomla! core icons."
 MOD_QUICKICON_GROUP_LABEL="Group"
 MOD_QUICKICON_INSTALL_EXTENSIONS="Install Extensions"
-MOD_QUICKICON_LANGUAGE_MANAGER="Language(s)"
+MOD_QUICKICON_LANGUAGE_MANAGER="Languages"
 MOD_QUICKICON_MAINTENANCE="Maintenance"
 MOD_QUICKICON_MEDIA_MANAGER="Media"
 MOD_QUICKICON_MENU_MANAGER="Menu(s)"
```
Comment (chmst) [inline administrator/language/en-GB/en-GB.mod_quickicon.ini:21]: Menus or Menu(s)?

**Comment by hardik-codes** [comment]
@brianteeman what all changes are to be done apart from the ones mentioned by @chmst

File: administrator/language/en-GB/en-GB.mod_quickicon.ini (hunk 1 above)
Comment (hardik-codes) [reply administrator/language/en-GB/en-GB.mod_quickicon.ini:21]: I think it should be Menus

**Comment by brianteeman** [comment]
i disagree because in all other cases there will always be a multiple eg plugins but in these by default there will only be one

**Comment by chmst** [comment]
@brianteeman you are right from the linguist's point of view. But for most users it "makes them think". In the Template group we have plural too, but only one template and style per default.

**Comment by hardik-codes** [comment]
@brianteeman and @chmst should I go about creating a new Pull Request  or leave it like that ?

**Comment by infograf768** [comment]
I agree with this change.
I see also that en-GB.mod_quickicon.ini needs care as a lot of strings are not used anymore in 4.0

imho should only remain
```
; Joomla! Project
; Copyright (C) 2005 - 2018 Open Source Matters. All rights reserved.
; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
; Note : All ini files need to be saved as UTF-8

MOD_QUICKICON="Quick Icons"
MOD_QUICKICON_ADD_NEW_ARTICLE="New Article"
MOD_QUICKICON_GLOBAL_CONFIGURATION="Global"
MOD_QUICKICON_GROUP_DESC="The group of this module (this value is compared with the group value used in <strong>Quick Icons</strong> plugins to inject icons). The 'mod_quickicon' group always displays the Joomla! core icons."
MOD_QUICKICON_GROUP_LABEL="Group"
MOD_QUICKICON_MEDIA_MANAGER="Media"
MOD_QUICKICON_MODULE_MANAGER="Modules"
MOD_QUICKICON_NAV_LABEL="Quick Links"
MOD_QUICKICON_XML_DESCRIPTION="This module shows Quick Icons that are visible on the Control Panel (administrator area home page)."

```

This means that the modifications in that file for these strings are useless as they should be deleted.

**Comment by chmst** [comment]
@hardik-codes you can commit into this branch, no need for a new pull request

**Comment by hardik-codes** [comment]
@infograf768 should I update the en-GB.mod_quickicon.ini in 4.0-dev as per above and then commit?

**Comment by brianteeman** [comment]
Got to love non English speakers 'correcting' English. That's why the English was so bad for so long.

PS there are two templates not one.

**Comment by hardik-codes** [comment]
@brianteeman what should I do then ?

**Comment by infograf768** [comment]
1. The (s) added in English is just unusable in many languages as they may have totally different words for plurals and single items. Therefore they have to choose and will use plural. 

2. It is much less confusing to use also plural in English as the word is in fact a shortcut when it was decided to take off the word "Manager" from the pages concerned.
For example "Languages" is for "Languages Manager"  where one can switch between site and admin (as for templates there are always a minimum of 2 languages...).
"Languages: Installed (Site)" and "Languages: Installed (Administrator)"
As for "Content Languages", there may indeed be only one, but it is also de facto a Manager that may contain multiple ones, title of page is "Languages: Content", not "Language(s): Content" or "Langages: Content(s)"

@hardik-codes 
Yes, I would also change the quickicons ini file in such a patch, but it can be done separately. In that case it is useless to modify its strings here.

**Comment by hardik-codes** [comment]
@infograf768 
In that case I should leave the en-GB.mod_quickicon.ini as it is and commit the rest. will that be OK ?

**Comment by brianteeman** [comment]
Other languages should localise. We should not go back to writing bad English so it is allegedly easier for a non native English speaker to translate. Or we can just go back to having ctap English and good French

**Comment by brianteeman** [comment]
This pr is wrong and should not be accepted

**Comment by infograf768** [comment]
@hardik-codes 
indeed. Useless to modify the quickicon file.

Generally speaking, 4.0 code leader will decide about this pr once tested and set rtc.

**Comment by infograf768** [comment]
And it is not bad English btw, it is just a difference of opinion concerning a label to navigate in the administration.

**Comment by hardik-codes** [comment]
Ok I got it

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 1593e332d5a1f82afc0a8408f173bfa030fad1bc<br /><br />Technically it is successfully tested. For me it looks good. 
As I am German I do not want to make a decision on English, but for foreigners it simply is easier.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23515">issues.joomla.org/tracker/joomla-cms/23515</a>.</sub>

**Comment by brianteeman** [comment]
> As I am German I do not want to make a decision on English, but for foreigners it simply is easier.

That is why we have translations. The english should not be altered to make it easier for non-english speakers

**Comment by infograf768** [comment]
The issue, again, is not about good English or not.
The issue is that we should keep the plural even if there may be only one item present, whether it is in English or not.

Look at the menus:
Do we write there Article(s) or Field(s) or User(s)

In system:
Do we use Installation Message(s) or Site Template(s) ?
Nope, we don't, and that is correct.

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 8db3b36f75fba11200bdcc9038cf08c116c3e060<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23515">issues.joomla.org/tracker/joomla-cms/23515</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23515">issues.joomla.org/tracker/joomla-cms/23515</a>.</sub>

**Comment by brianteeman** [comment]
Please do not merge this - it is wrong

**Comment by Joomv** [comment]
I am sorry but this change is not the right use of en-gb.
The parentheses should stay.

**Comment by Joomv** [comment]
We refer to articles, modules, components, users in the plural because there is always more than one installed with Joomla and in most cases always more than one user. 
The same cannot be said for languages.  Therefore brackets should stay.

**Comment by infograf768** [comment]
>  and in most cases

"Most" is the key word here...
Again, as title says, the use of plurals with parenthesis in all these navigation labels is totally inconsistent. Or you add them everywhere (Except obviously for Extensions, Modules, Components. I am forgetting some) or you don't for any and use plurals.

and Also for the Managers page titles...

From
screen shot 2019-01-17 at 10 25 12

To
screen shot 2019-01-17 at 10 29 14


To be a native English speaker is not a proof that using (s) is correct in these cases. It is just a matter of logic and consistency

**Comment by Joomv** [comment]
But only for users all other items mentioned have more than one of them.
This is a ridiculous amount of time being wasted over semantics.  
The brackets for languages should stay as it is correct in the context.

**Comment by Bakual** [comment]
Most of my sites only have one user (most personal sites will). Yet we write "Users".
All non-english sites will have two languages (en-GB and the localised one), probably also two content languages except if they deleted the default english one.

I agree with the OP and JM here that plural makes more sense. At least it is consistent across the CMS then.
It makes absolutely no sense to make an arbitrary choice of menu items use the (s) while in other places where as well may be a singular item in it, we use the plural forms (users, articles, categories, banners, ...).
Beside that it looks pretty stupid to me. And don't tell me that's because I'm not native english. We can do that in german as well (Sprache(n), Menü(s)) and it looks stupid here as well and is thus rarely used. Plural form works absolutely fine.
But at least do it consistent across the board.

**Comment by brianteeman** [comment]
What a shame. The only native English speakers commenting on an issue about the use of English in joomla are completely ignored. Why even bother opening a pull request. You might as well just merge directly if people are not interested in and do not value the legitimate comments of those that speak the language.

**Comment by rdeutz** [comment]
Comments were not ignored, you need to understand that even you disagree that the final decision can be different.

**Comment by brianteeman** [comment]
When it  comes to the use of english and the only english speakers say it is wrong and it still merged then they were clearly ignored

**Comment by Joomv** [comment]
I'm sorry but this is still wrong. 
It may not be the way to do things in other languages, but it is in English.
Plus there's a marketing angle to it - English is arrogant thinks it is the only language - having the (s) prompts and reminds English speakers there's more than one language and also that we do multi-lingual.
How a group of non-natives can just carte blanche decide what to do with English because it suits their language is just beyond me.

**Comment by brianteeman** [comment]
@joomv I have battled this for 13 years

**Comment by infograf768** [comment]
As we have demonstrated, the alternative is to use (s) almost in every case to be consistent, which makes absolutely no sense for our Interface, in English as well as any other language.

Read the comments here by English natives:
https://english.stackexchange.com/questions/93940/number-agreement-when-using-s-for-optional-plural 

"Use of (s) might be necessary in legal documents, which have to be very precise and cover all the possible meanings. In ordinary writing, in graphical interfaces, and in technical documentation, I don't think it's necessary at all. I suspect that programmers want to be more precise than is necessary when they use this form in graphical interfaces. As a technical writer, I've always used the plural, and recommended it in our graphical interfaces."

"Using the plural form "Selecting appropriate topics" doesn't preclude the possibility of finding only one appropriate topic--or even none."

In our case it would be "Topics".
And this one is rightfully now closed as obviously Joomla is not a legal document and should have a clean readable interface, even in English...

I am now out of this useless discussion.

**Comment by brianteeman** [comment]
> I am now out of this useless discussion.

useless to you but not to the english speakers!!

**Comment by Bakual** [comment]
I would understand your point if it was consistent across the CMS, but it wasn't. It was just some random strings which had the (s).
Now it's at least consistent. If you think it's wrong, do a PR and change **all** instances to use an (s), not just a few.

**Pull request merged**